Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,129 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            //declare my variables
+            string firstName;
+            string lastName;
+            float age;
+            float weightInKg;
+            float height;
+
+            //ask for first input of name
+            // then send call method to make sure it fits name paramter of only letters and assign to name variable
+            Console.WriteLine("Please enter your first name.");
+            firstName = CheckStringForLettersOnly();
+
+            // same as first name but for last name
+            Console.WriteLine("Please enter your last name.");
+            lastName = CheckStringForLettersOnly();
+
+
+            // same as name inputs but for age
+            //call age method and check to make sure it fits parameter of numbers only.
+            Console.WriteLine("Please enter your age.");
+            //I went with float for age because maybe someone likes to put percent of new age such as 7.5 years old...
+            age = float.Parse(CheckStringForDigitsOnly());
+
+            //same as age
+            Console.WriteLine("Please enter your weight in kg.");
+            //weightInKg = float.Parse(CheckStringForDigitsOnly(weightInKg.ToString()));
+            weightInKg = float.Parse(CheckStringForDigitsOnly());
+
+            //same as age
+            Console.WriteLine("Please enter your height in cm.");
+            height = float.Parse(CheckStringForDigitsOnly());
+
+
+            //after all values added run this at end of program
+            Console.WriteLine("Thank you for your information. You entered:");
+
+            //now dispaly the data that was given in a sentence. 
+            Console.WriteLine($"\n``` {firstName} {lastName} is {age} years old, your weight is {weightInKg} kg and" +
+                              $" your height is {height} cm. ```");
+
+            //tell user to press anybutton to close
+            Console.WriteLine("Press any button to exit.");
+
+            //wait for the button to be pressed
+            Console.ReadKey();
+        }
+
+        //Method that checks if string has only letters in it
+        private static string CheckStringForLettersOnly()
+        {
+            bool stop = true;
+            string testString = "";
+
+            while (stop)
+            {
+                //read the input from console this is what will be tested for only letters
+                testString = Console.ReadLine();
+
+                //check it by calling the method IsAllLetters
+                // if it has a number run this 
+                if (IsAllLetters(testString) != true)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4837469)